sphinx/util/i18n.py

```diff
@@ -67,15 +67,15 @@ def write_mo(self, locale):
         with io.open(self.po_path, 'rt', encoding=self.charset) as file_po:
             try:
                 po = read_po(file_po, locale)
-            except Exception:
-                logger.warning('reading error: %s', self.po_path)
+            except Exception as ex:
```

Comment:
In this project, we usually use `exc` or `err` for variable name. Could you rename this to `exc` please?